src/main/java/org/kumoricon/registration/staff/staffimport/StaffImportFile.java

```diff
@@ -175,23 +176,60 @@ public String toString() {
+                        "RPG", "Idol Festival", "LARP", "PC Gaming", "Live Events", "Tabletop Gaming", "Escape Room",
+                        "Video Gaming", "Fan Fiction", "Play and Win", "Gunpla Lounge", "Pokémon Gaming Assistant Coordinator",
+                        "Programming", "Programming Office", "TCG", "Lighting Equipment", "Video Equipment", "Programming Booth",
+                        "Pokémon Gaming"),
+                "Publicity", List.of("Hall Cosplay", "Info Booth", "Software", "Merchandise", "Press", "Social Media",
+                        "Website", "Marketing", "Newsletter", "Multimedia", "Graphics", "Publicity"),
+                "Relations", List.of("Relations", "Autographs", "Relations Logistics", "Hospitality", "Industry and Sponsorship",
+                        "Guests"),
+                "Operations", List.of("Hotel Attendee Services", "Attendee Services", "Operations Office", "Accessibility",
+                        "Cosplay Repair", "Operations"),
+                "Treasury", List.of("Treasury", "Supply and Logistics"),
+                "Secretarial", List.of("Secretary"),
+                "Chair", List.of("Chair", "EDI", "Staff Retention", "Internal Support", "Outreach", "Staff Events",
+                        "Training and Development", "Charity", "External Support", "Art Show", "Photobooth", "Nonprofit Grants")
+        );
 
         @JsonCreator
         public Position(
                 @JsonProperty(value = "term", required = true) String term,
-                @JsonProperty(value = "titleBadgeShort", required = true) String title,
-                @JsonProperty(value = "rank", required = true) String rank,
-                @JsonProperty(value = "department", required = true) String department,
-                @JsonProperty(value = "departmentSuppressed", required = true) Boolean departmentSuppressed) {
+                @JsonProperty(value = "team", required = true) String team,
+                @JsonProperty(value = "title", required = true) String title,
+                @JsonProperty(value = "level", required = true) String level,
+                @JsonProperty(value = "isElected", required = true) Boolean isElected) {
             this.term = term;
+            this.team = team;
             this.title = title;
-            this.rank = rank;
-            this.department = department;
-            this.departmentSuppressed = departmentSuppressed;
+            this.level = level;
+            this.isElected = isElected;
+            this.department = getDepartmentFromTeam(team);
+        }
+
+        private String getDepartmentFromTeam(final String team) {
+            for (final String dept : DEPARTMENT_TEAMS.keySet()) {
+                if (DEPARTMENT_TEAMS.get(dept).contains(team)) {
+                    return dept;
+                }
+            }
+            System.out.println(team);
+
+            return "Membership";
```

Comment:
fuck sorry this was leftover from some testing 

this pr actually isnt ready to merge until i hear back about some teams whose corresponding department is unknown currently